lib/traces/backend/open_telemetry/interface.rb

```diff
@@ -66,13 +66,18 @@ def trace_context=(context)
 				end
 				
 				def trace_context(span = ::OpenTelemetry::Trace.current_span)
+					# state = baggage.values(context: span.context)
 					if span_context = span.context
-						state = baggage.values(context: span.context)
+							flags = 0
+							
+							if span_context.trace_flags.sampled?
+								flags |= Context::SAMPLED
+							end
```

Comment:
I pulled this from the Datadog Backend. as it was implemented, the trace flags were expected to be provided as an integer, not an object.